File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 3/ex15.c
#include <stdio.h>

main()
{
    printf("Numero Primo\n\n");

    //vars
    int n, divisor, cont=0;

    //Entrada
    printf("Insira um numero positivo: ");
    scanf("%d",&n);

    //Processamento
    for(divisor=1; divisor<=n; divisor++)
    {
        if( n%divisor == 0 )
            cont++;
    }

    //Saída
    if( cont == 2 )
        printf("%d eh primo", n);
    else
        printf("%d NAO eh primo", n);

}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 3/src/Ex15.java
//Importando Scanner
import java.util.Scanner;

public class Ex15 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Número Primo\n\n");
		
		//vars
		int n, divisor, cont=0;
		
		//Entrada
		System.out.print("Digite um número positivo: ");
		n = ent.nextInt();
		
		//Processamento
		for ( divisor=1; divisor<=n; divisor++ )
		{
			if( n%divisor == 0 )
				cont++;
		}
		
		//Saída
		if ( cont == 2 )
			System.out.println(n + " é primo!");
		else
			System.out.println(n + " NÃO é primo!");
		
	}
}
